Documentation/build-container-examples/play-example/build-play-example.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$EUID" -ne 0 ]; then
+    echo "This script uses functionality which requires root privileges"
+    exit 1
+fi
+
+# Start the build with an empty ACI
+acbuild --debug begin docker://ubuntu:17.10
+
+# In the event of the script exiting, end the build
+trap "{ export EXT=$?; acbuild --debug end && exit $EXT; }" EXIT
+
+# Name the ACI
+acbuild --debug set-name example.com/play-example
+
+# Install java, psql, and git
+acbuild --debug run -- apt-get update
+acbuild --debug run -- apt-get -y install openjdk-8-jre openjdk-8-jdk git postgresql
```

Comment:
could we uninstall the unneeded components after they are used in the build process? Are `git` and `postgresql` still needed at run-time?

Reply:
oh, I see that the client side `psql` is used..

Reply:
I can uninstall git